package.json

```diff
@@ -38,7 +38,7 @@
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "date-fns": "^1.29.0",
-    "design-system": "git://github.com/Wikia/design-system.git#21.10.0",
+    "design-system": "git://github.com/Wikia/design-system.git#8711cc66614a035c436d4eaa42e7c8d88b5f8d9d",
```

Comment:
Yes, exactly. I'm not sure what the final DS version number will be, because there may be some other release before mine. I'll update it once I release DS.